python/pyfeelpp-toolboxes/feelpp/toolboxes/fluid/fluid.cpp

```diff
@@ -56,9 +56,11 @@ void defFM(py::module &m)
              )
         .def("init",static_cast<void (fm_t::*)(bool)>(&fm_t::init), "initialize the fluid mechanics toolbox",py::arg("buildModelAlgebraicFactory")= true)
         .def("mesh",static_cast<typename fm_t::mesh_ptrtype  (fm_t::*)() const>(&fm_t::mesh), "get the mesh")
+        .def("rangeMeshElements", &fm_t::rangeMeshElements, "get the range of mesh elements" )
         .def("setMesh",static_cast<void (fm_t::*)(typename fm_t::mesh_ptrtype const&)>(&fm_t::setMesh), "set the mesh of the toolbox",py::arg("mesh"))
+        .def("updateParameterValues", &fm_t::updateParameterValues, "update parameter values" )
         // function spaces and elements
-        .def("functionSpaceVelocity",&fm_t::functionSpaceVelocity, "get the velocity function space")
+        .def("spaceVelocity",&fm_t::functionSpaceVelocity, "get the velocity function space")
```

Comment:
@thomas-saigre @alielarif why the change of name ?

Reply:
That was after a conversation we had on slack : https://feelpp.slack.com/archives/C6A375EMS/p1662299935892779 :

> @prudhomm juste une petite remarque pour le wrapping python : dans la toolbox heat, pour accéder aux fonction de l'espace, la fonction est spaceTemperature(), tandis que pour la toolbox fluid c'est functionSpaceVelocity(). Est-ce que ce ne serait pas plus cohérent de mettre le même template de nom pour toutes les toolbox (en plus ça perd github copilot !)